src/dependency-trace.ts

```diff
@@ -13,11 +13,20 @@ export interface DependencyTrace {
  * inspected.
  */
 export const dependencyTrace = (context: string, name: string, model: {}, message = {}, relay = {}) => {
+
+  /**
+   * Symbols cannot cross the `eval` boundary, so they need to be coerced to
+   * their string equivalent.
+   */
+  const sanitizeKey = (key: any) =>
+    typeof key === 'symbol' ? key.toString() : key;
```

Comment:
If we're only using this for paths, this isn't going to be a useful implementation, because the only symbol we use, `PARENT`, is intended to walk back up the path, so with this, the value of `fullPath` below is going to return an invalid path.

Reply:
I've tested this with a container that uses `delegate: PARENT` and it still works as expected; Casium explicitly excludes the `PARENT` symbol when constructing `path`: https://github.com/ai-labs-team/casium/blob/master/src/runtime/exec_context.ts#L169

Reply:
Oh, huh, never mind then.